plugins/reach/lib/model/data/VendorModerationCatalogItem.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * @package plugins.reach
+ * @subpackage model
+ */
+class VendorModerationCatalogItem extends VendorCatalogItem
+{
+    public function applyDefaultValues(): void
+    {
+        $this->setServiceFeature(VendorServiceFeature::MODERATION);
+    }
+
+    public function isDuplicateTask(entry $entry): bool
+    {
+        return false;
+    }
+
+    public function isEntryTypeSupported($type, $mediaType = null): bool
```

Comment:
This overrides a similar function, which also checks for additional types from plugins.
Unless there is a reason to ignore mediaEntry of type externalMedia - you don't need to override it.

Reply:
removed. tnx